### ISSUE TITLE
FR: Plugin instructions on the config page

### ISSUE THREAD

**Issue by profucius** [ohm-en/obsidian-plugin-manager#8]
I propose a section at the very top of the plugins list in your config page, which explains what the user should be doing while on the config page. Currently there is just a list of plugins with toggle buttons and a box with numbers in them. A user who just installed the plugin will remember what to do after having read the instructions, but months from now may not remember what to do, what the numbers mean, etc.

Perhaps remind the user that the number box is how many seconds they want the plugin to be loaded after Obsidian loads, and the toggle box is for enable/disabling the lazy-load.

And perhaps leave a link to your github there too, so that power users can visit to read more, see changelogs, etc.

**Comment by ksdavidc** [comment]
agreed. It took me a while to find out that the setting represents seconds, not milliseconds.
